google/install-chrome/README.md

```diff
@@ -77,5 +77,120 @@ The following output values are available:
 - `${{ tasks.chrome.values.chrome-directory }}`
 - `${{ tasks.chrome.values.chromedriver-binary }}`
 - `${{ tasks.chrome.values.chromedriver-directory }}`
+- `${{ tasks.chrome.values.start-virtual-display-binary }} `
+- `${{ tasks.chrome.values.is-virtual-display-running-binary }} `
```

Comment:
These both have a trailing space before the closing tick.

Reply:
Thanks!